docs/content/en/docs/implementing/tasks-non-k8s-apps.md

```diff
@@ -65,21 +97,33 @@ spec:
     workloadVersion: "1.0.0"
 ```
 
-Applying this file will cause Keptn to create a Job and a Pod and run the `KeptnTaskDefinition`.
+You can then apply this YAML file with the following command:
+
+```shell
+kubectl apply -f test-task.yaml -n my-keptn-annotated-namespace
+```
+
+Applying this file causes Keptn to create a Job and a Pod
+and run the executables defined
+in the associated `KeptnTaskDefinition` resource.
 
-`kubectl get keptntasks` and `kubectl get pods` will show the current status of the jobs.
+Use the following commands to show the current status of the jobs:
 
-## Running More KeptnTasks
+```shell
+kubectl get keptntasks -n my-keptn-annotated-namespace
+kubectl get pods -n my-keptn-annotated-namespace
+```
 
-For subsequent KeptnTask runs, the `KeptnTask` name needs to be unique, update the follow fields:
+## Re-run the KeptnTask
 
-- `name`
-- `spec.appVersion`
-- `spec.workloadVersion`
-- `spec.context.appVersion`
-- `spec.context.workloadVersion`
+For subsequent KeptnTask runs,
+the `KeptnTask` name and version fields must be unique,
+so copy the `KeptnTask` yaml file you have and update the
+`metadata.name` field.
 
-For example:
+A standard practice is to just increment the value of the suffix field.
+For example, you could create a `test-task-2.yaml` file
+with the `metadata.name` field set to `runhelloworld2`:
 
 ```yaml
```

Comment:
I know this is not modified here , but should we profit from the fact that we touch this doc to embed the yaml instead? 
We could edit [this sample](https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/config/samples/lifecycle_v1alpha3_keptntask.yaml) and point to it

Reply:
I think we should merge this PR -- it's more than a month old -- then do another PR to make this change.  @mowies complains about the tendency to do one more thing while the file is open...